### ISSUE TITLE
Follow up on CNN deprecation done in the update to python 3.10.

### ISSUE THREAD

**Issue by samuelklee** [broadinstitute/gatk#8907]
After the updates to the python environment made in #8561, the CNN tools are no longer functional and were deprecated.

However, we may want to provide a separate environment in which some of these tools can still be used, perhaps in conjunction with NVScoreVariants. If so, we should also re-enable the relevant integration and WDL tests and provide appropriate documentation.

**Comment by droazen** [comment]
Yes, the plan is to distribute the old environment as just a yml file that can be optionally activated -- provided that we can set up part of the test suite to run under the old environment, which is the next task.
